tests/index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
```

Comment:
Remove this file if it is not used for running tests

Reply:
It just a test coverage result, removed.